packages/interaction-kit/src/application.ts

```diff
@@ -65,6 +69,14 @@ export default class Application {
 			throw new Error("Please provide a Token. You can find this value <here>");
 		}
 
+		if (autoDeferApplicationCommands) {
+			this.#autoDeferApplicationCommands = autoDeferApplicationCommands as boolean;
+		}
+
+		if (autoDeferMessageComponents) {
+			this.#autoDeferMessageComponents = autoDeferMessageComponents as boolean;
+		}
+
```

Comment:
```suggestion
		this.#autoDeferApplicationCommands = !!autoDeferApplicationCommands;

		this.#autoDeferMessageComponents = !!autoDeferMessageComponents;

```

Reply:
What does !! mean?

Reply:
> What does !! mean?

`!!expression` will convert the expression into a boolean. That ensures there's no unexpected behavior if a different type gets set.

```ts
!!0; // false
!!1; // true
!!null; // false
!!{}; // true
```

Reply:
Oh Sweet